analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
About, the commit message, I get it. But do you think it is worth adding how `FastForest` is supposed to work in README rather than in the commit message body?

Reply:
yes absolutely, good idea!

Reply:
Also, `fastforest` is supposed to be installed as it is used as an external library. Can we add fastforest directory containing headers and libs directly to the repo? Or do we leave it for a user to care about library installation?

Reply:
we leave it to users, we can't add the compiled binaries of FastForest, they will not work on every platform -- users have to install FastForest on their own. we can provide instructions for our side of things though.

Reply:
I've updated the commit message. Should it be more detailed than now?

Reply:
It's perfect, thanks!